bin/filter_gtf_for_genes_in_genome.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
Does this script come from somewhere, or is it your implementation? 

In any case I'd add a small comment who wrote it just in case someone has a question related to it.

Reply:
Attribution now included. Good suggestion, thanks!